src/components/Utils/getInventoryRecordsTypeForSelect.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { INVENTORY_RECORDS_TYPE } from '../POLine/const';
+
+const getInventoryRecordsTypeForSelect = () => (
```

Comment:
as for me it should be separate component that can be used on any forms

Reply:
Updated